docs/glossary.md

```diff
@@ -0,0 +1,468 @@
+
+  :::{seealso}
+
+  - <https://docs.epics-controls.org/en/latest/appdevguide/gettingStarted.html#building-ioc-components>
+
+  :::
+
+**Record**
+  EPICS-based control system contains one or more IOC.
+  Each IOC loads one or more databases.
+  A database is a collection of records of various types.
+
+  A Record is an object with:
+
+  - A unique name.
+  - A behavior defined by its type.
+  - Controllable properties (**fields**).
+  - Optional associated hardware I/O (device support).
+  - Links to other records.
+
+  There are several different types of records available. For example:
+
+  - The "analog input" and "analog output" (`ai` and `ao`) types,
+    are used to store an analog value,
+    and are typically used for things like temperatures, pressure, flow rates, etc.
+  - The "binary input" and "binary output" (`bi` and `bo`) types,
+    are used to store a boolean value,
+    and are generally used for commands and statuses to and from equipment,
+    i.e. for values like On/Off, Open/Closed and so on.
+  - The "calc" and "calcout" records can access other records
+    and perform a calculation based on their values.
+    E.g. calculate the efficiency of a motor
+    by a function of the current and voltage input and output,
+    and converting to a percentage for the operator to read.
+
+  Each record comprises a number of properties called fields.
+  Fields can have different functions,
+  typically they are used to configure how the record operates,
+  or to store data items.
+
+  :::{seealso}
+
+  - <https://docs.epics-controls.org/en/latest/process-database/EPICS_Process_Database_Concepts.html?highlight=PV#the-epics-process-database>
+  - <https://docs.epics-controls.org/en/latest/appdevguide/databaseDefinition.html?highlight=record#record-record-instance>
+
+  :::
+
+**PV**
+  A Process Variable is a value or variable accessible through EPICS communication protocols
+  (Channel Access and PV Access).
+  You can address this value using its unique PV name.
+
+  In the technical sense,
+  a PV is an "instantiated"/"implemented" record.
+  I.e. once the IOC program is started,
+  each previously defined record will have an associated PV running.
+  A Record is just a blueprint for a PV.
+
+  :::{seealso}
+
+  - <https://docs.epics-controls.org/en/latest/internal/ca_protocol.html?highlight=Process%20Variable#process-variables>
+  - <https://docs.epics-controls.org/en/latest/process-database/EPICS_Process_Database_Concepts.html?highlight=Process%20Variable>
+  - <https://docs.epics-controls.org/en/latest/process-database/EPICS_Process_Database_Concepts.html?highlight=Process%20Variable#process-chains>
+
+  :::
+
+**Macro**
+  A macro is a string substitution mechanism,
+  that will allow some EPICS "configuration" files to be loaded
+  after some strings have been replaced by others.
+  E.g. `MY-MACRO-NAME=toto`, will replace every `${MY-MACRO-NAME}` by `toto`
```

Comment:
I'm not sure whether `-` are allowed in macro names, but it seems the underscore is more popular.

```suggestion
  E.g. `MY_MACRO_NAME=toto`, will replace every `${MY_MACRO_NAME}` by `toto`
```